controller/workitem.go

```diff
@@ -634,8 +634,11 @@ func ConvertWorkItem(request *goa.RequestData, wi workitem.WorkItem, additional
 			}
 
 		case workitem.SystemTitle:
-			// 'HTML escape' the title to prevent script injection
-			op.Attributes[name] = html.EscapeString(val.(string))
+			if val != nil {
+				op.Attributes[name] = val.(string)
```

Comment:
thanks @sbose. I addressed this safety here and in all similar casts in this function, too ;)